io/plugins/eu.esdihumboldt.hale.io.jdbc.msaccess.test/.settings/org.eclipse.jdt.core.prefs

```diff
@@ -0,0 +1,7 @@
+eclipse.preferences.version=1
```

Comment:
You did not run the `apply-preferences` script yet for the new projects, please do so.